core/chains/evm/logpoller/orm.go

```diff
@@ -73,8 +73,12 @@ func (o *ORM) DeleteFilter(name string, qopts ...pg.QOpt) error {
 func (o *ORM) LoadFilters(qopts ...pg.QOpt) (map[string]Filter, error) {
 	q := o.q.WithOpts(qopts...)
 	rows := make([]Filter, 0)
-	err := q.Select(&rows, `SELECT name, ARRAY_AGG(DISTINCT address)::BYTEA[] AS addresses, ARRAY_AGG(DISTINCT event)::BYTEA[] AS event_sigs
-									FROM evm_log_poller_filters WHERE evm_chain_id = $1 GROUP BY name`, utils.NewBig(o.chainID))
+	err := q.Select(&rows, `SELECT name,
+										ARRAY_AGG(DISTINCT address)::BYTEA[] AS addresses, 
+										ARRAY_AGG(DISTINCT event)::BYTEA[] AS event_sigs,
+										MAX(retention) AS retention
+									FROM evm_log_poller_filters WHERE evm_chain_id = $1
+									GROUP BY name`, utils.NewBig(o.chainID))
```

Comment:
This formatting is weird

Reply:
Oh, my GoLand settings are 4 spaces per tab, here it's 8. Are we supposed to be using 8?